endoreg_db/utils/pipelines/process_video_dir.py

```
@@ -0,0 +1,120 @@
+from endoreg_db.models import (
+    VideoFile, Center, EndoscopyProcessor, AiModel, ModelMeta,
```

Ruff (0.8.2)

`endoreg_db.models.Center` imported but unused

Remove unused import

(F401)